token.json

```diff
@@ -0,0 +1 @@
+{"access_token":"ya29.Gls6BudTLH6IErPdUS1xPQMPUOPwvJVwneAYO4j0PIG4b-yewUeOJllQszAs_QuhKB_VT4TeSGQhke6boSA8Zq_pdMUP6Qgh-4xaC6ix5W4qxNfYlj-Ob4OI1zm-","refresh_token":"1/2efG7YHnZLdrN_Oy3NeaPKMU1aEPWezij9e4n4yo0g4","scope":"https://www.googleapis.com/auth/drive.metadata.readonly","token_type":"Bearer","expiry_date":1539853462716}
```

Comment:
Don't add tokens directly as commit. Take it from environment or in this case, `config.js` which won't be committed.

Remove `token.json` file.

Reply:
`token.json` will be created when you'll use your credentials for google drive for the first time, after you'lll enter a one time access code generated by going to a url. 

adding it to .gitignore so that it won't be commited.

Reply:
It's not mandatory to create `token.json`
You can place the token in other places or on client side as well. Figure out a way to NOT use `token.json`

Also, adding to `.gitignore` won't delete this just yet. You've already committed this.